crates/flox/src/commands/environment.rs

```diff
@@ -479,49 +440,49 @@ pub struct Uninstall {
     environment_args: EnvironmentArgs,
 
     #[bpaf(external(environment_select), fallback(Default::default()))]
-    environment: EnvironmentSelect,
+    _environment: EnvironmentSelect,
 
     #[bpaf(positional("PACKAGES"), some("Must specify at least one package"))]
-    packages: Vec<String>,
+    _packages: Vec<String>,
 }
 
 impl Uninstall {
-    pub async fn handle(self, flox: Flox) -> Result<()> {
+    pub async fn handle(self, _flox: Flox) -> Result<()> {
         subcommand_metric!("uninstall");
 
-        let packages: Vec<_> = self
-            .packages
-            .iter()
-            .map(|package| FloxPackage::parse(package, &flox.channels, DEFAULT_CHANNEL))
-            .collect::<Result<Vec<_>, _>>()?
-            .into_iter()
-            .dedup()
-            .collect();
-
-        let mut environment = self
-            .environment
-            .to_concrete_environment(&flox)?
-            .into_dyn_environment();
-
-        let packages_str = packages_to_string(&packages);
-        let plural = packages.len() > 1;
-
-        if environment
-            .uninstall(packages, &flox.nix(Default::default()), flox.system)
-            .await
-            .context("could not uninstall packages")?
-        {
-            info!(
-                "🗑️ Uninstalled {packages_str} from '{}' environment.",
-                environment.environment_ref()
-            );
-        } else {
-            let verb = if plural { "are" } else { "is" };
-            info!(
-                "No changes; {packages_str} {verb} not installed in '{}' environment.",
-                environment.environment_ref()
-            );
-        }
+        // let packages: Vec<_> = self
```

Comment:
Also here, why have so much commented out?

Reply:
See the comment above.